### PR TITLE
Merge report_lin to master

### PR DESCRIPTION
**Pull request by cameron93lin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 ## DESCRIPTION
 
-## Environment Setup
+#### Environment Setup
+This project is training and testing on Ubuntu 18.04 with Anaconda. We use one Quatro RTX 6000 and two RTX 2070 Super to train our model. Because all the video cards we use are RTX series, we can enable the RTX 16-bit optimization which makes our training faster. The model is built on Keras. The training is on blocks and patches. We use three framesets at a time, which is 9 frames in total. The block memory we use is about 22 GB and patch memory is about 6 GB.
```
Comment (Chlerry) [inline README.md:6]: We train ... 
... The model is built on Keras... 
The training is on blocks and patches. 
frame sets
which have 
why 22GB and 6GB: 1 pixel -> block & patch

Comment (Chlerry) [reply README.md:6]: mini-batch

File: README.md [hunk 2]
```diff
@@ -2,7 +2,18 @@
...
+
+For the testing data, we use the triplet dataset and HEVC dataset. For the HEVC dataset, we use BlowingBubbles and BasketballDrill. Each part has 500 frames and we use 250 of these 500 frames for testing.
+
+#### Training method/configuration 
+To train our neural network, we initialized our neural network parameters using Xavier initialization method and use the AdaMax to optimize the proposed network. Xavier initialization is a method to ensure variance of both input and output to be the same. We set beta1 to 0.9 and beta2 to 0.999. The learning rate is 0.001 and we use 128 for the Mini-batch size to minimize the loss function. The maximum epochs number is set to 1000.
```
Comment (Chlerry) [inline README.md:14]: beta1, beta2, and learning rate are parameters of AdaMax

Comment (Chlerry) [reply README.md:14]: AdaMax optimizer

File: README.md [hunk 3]
```diff
@@ -2,7 +2,18 @@
...
+
+We also use EarlyStopping to increase our training speed. We monitor validation loss with 1.0 for Min_delta and 10 for the patience. The model will save the best one to hdf5 file when early stopping is occurred.
```
Comment (fenyangwang) [inline README.md:16]: is occurred -> is triggered.
